src/hooks/useTimer.ts

```diff
@@ -1,13 +1,21 @@
 import { useEffect, useRef, useState } from "react";
 
 export const useTimer = (
-  isActive: boolean,
   onResetTimer: () => void,
   initialSeconds: number = 180
 ) => {
+  const [isTimerActive, setIsActive] = useState(false);
   const [timeLeft, setTimeLeft] = useState(initialSeconds);
   const timerRef = useRef<NodeJS.Timeout | null>(null);
```

Comment:
정말 사소한 것이지만 브라우저의 setTimeout과 `NodeJS.Timeout`의 반환 타입이 다른 것으로 알고 있어요. 실제 저희가 사용하는 것은 프론트엔드의 브라우저 setTimeout 이고, NodeJS 타입을 사용하는 것은 어색하다고 생각해요.

```suggestion
  const timerRef = useRef<ReturnType<typeof setTimeout>>();
```
으로 쓰는게 더 좋을 것 같아요.

Reply:
헉 체크 감사드립니다!